applications/DEM_application/tests/test_analytics.py

```diff
@@ -60,55 +59,55 @@ def FinalizeTimeStep(self, time):
                     self.CheckValueOfNormalImpactVelocity(normal_impact_vel, expected_value, tolerance)
                     expected_value = 0.0
                     self.CheckValueOfFaceNormalImpactVelocity(face_normal_impact_vel, expected_value, tolerance)
-                elif time > 0.04 and time < 0.28:
+                elif time > 0.045 and time < 0.28:
                     expected_value = 3.0
                     self.CheckValueOfNormalImpactVelocity(normal_impact_vel, expected_value, tolerance)
```

Comment:
unrelated but I think you could use the json_check_process from the core, then you don't have to hardcode it

Reply:
Agree